Controller/ContentManagement/DataController.php

```diff
@@ -32,12 +32,15 @@
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\ParamConverter;
 use Symfony\Component\Form\Extension\Core\Type\SubmitType;
 use Symfony\Component\Form\FormError;
+use Symfony\Component\HttpFoundation\RedirectResponse;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\HttpFoundation\ResponseHeaderBag;
 use Symfony\Component\HttpKernel\Exception\BadRequestHttpException;
 use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
 use Symfony\Component\Routing\Annotation\Route;
+use Throwable;
```

Comment:
Use \ to prefix Standard Library functions & classes for performance gain in the code, and remove the use Throwable statement.
See point 3 at https://dev.to/elabftw/optimizing-your-php-app-speed-3hd4 

Symfony does the  same.

Reply:
see 405d08581febb1c8c36f03488d1b3623e8899e83